### PR TITLE
Fix/ui streams creation ux

### PR DESCRIPTION
**Pull request by ajainarayanan**
Fixes creating streams in UI.
- Shows loading when creating a stream.
- Shows error message in context.
- Show success page.
- Fixes pagination in datasets, apps & runs history view
- Fixes search in pagination

### CODE REVIEW

**Comment by ajainarayanan** [comment]
Cluster: http://nodeproxy1736-1000.dev.continuuity.net:9999/
streamcreation_pagination

File: cdap-ui/app/features/admin/controllers/NamespaceCreateController.js [hunk 1]
```diff
@@ -14,19 +21,28 @@ angular.module(PKG.name + '.feature.admin')
...
-        });
-    });
+            myNamespace.getList(true).then(function() {
+              EventPipe.emit('namespace.update');
+            });
+            queryInProgress = false;
+          },
+          function error(err) {
+            $scope.isSaving = false;
+            queryInProgress = false;
```
Comment (elfenheart) [inline cdap-ui/app/features/admin/controllers/NamespaceCreateController.js]: consolidate queryInProgress with $scope.isSaving?

**Comment by elfenheart** [comment]
:+1: LGTM

**Comment by anwar6953** [comment]
+1

**Pull request merged**